whois-xml-api/info.json

```diff
@@ -0,0 +1,979 @@
+      "enabled": true,
+      "category": "investigation",
+      "annotation": "reverse_dns_search",
+      "output_schema": {
+        "current_page": "",
+        "size": "",
+        "result": [
+          {
+            "name": "",
+            "first_seen": "",
+            "last_visit": ""
+          }
+        ]
+      },
+      "parameters": [
+        {
+          "title": "DNS Type",
+          "name": "reverse_dns_type",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "select",
+          "options": [
+            "IPv4 Address",
+            "Mail Server",
+            "Name Server"
+          ],
+          "onchange": {
+            "IPv4 address": [
+              {
+                "title": "IPv4 Address",
+                "name": "ip",
+                "visible": true,
+                "required": true,
+                "editable": true,
+                "type": "text",
+                "placeholder": "8.8.8.8",
+                "tooltip": "Retrive lists all domains that have resolved to a given IP address at one point in time on the Domain Name System (DNS)",
+                "description": "Retrive lists all domains that have resolved to a given IP address at one point in time on the Domain Name System (DNS)"
+              }
+            ],
+            "Mail server": [
+              {
+                "title": "Mail server",
+                "name": "mx",
+                "visible": true,
+                "required": true,
+                "editable": true,
+                "type": "text",
+                "placeholder": "mx.google.com",
+                "tooltip": "Retrive lists all domains that have resolved to a given mail server at one point in time on the Domain Name System (DNS)",
+                "description": "Retrive lists all domains that have resolved to a given mail server at one point in time on the Domain Name System (DNS)"
+              }
+            ],
+            "Name server": [
+              {
+                "title": "Name server",
+                "name": "ns",
+                "visible": true,
+                "required": true,
+                "editable": true,
+                "type": "text",
+                "placeholder": "ns2.google.com",
+                "tooltip": "Retrive lists all domains that have resolved to a given name server at one point in time on the Domain Name System (DNS)",
+                "description": "Retrive lists all domains that have resolved to a given name server at one point in time on the Domain Name System (DNS)"
+              }
+            ]
+          },
+          "tooltip": "Select the method from the following options using which you want to retrive reverse dns",
+          "description": "Select the method from the following options using which you want to retrive reverse dns"
+        },
+        {
+          "title": "From",
```

Comment:
can we have better name like Retrieve Results from Domain or From Domain?

Reply:
done